.travis.yml

```diff
@@ -1,44 +0,0 @@
-language: php
```

Comment:
are you sure you want to remove the travis file? i propose that we leave it, even if there are no tests at the moment.